apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -411,6 +411,7 @@ export default Vue.extend({
       //   const focusingTable = this.tabulator.element.contains(document.activeElement)
       //   if (!focusingTable) this.page--
       // }
+      result['alt+m'] = this.openEditorMenuByShortcut.bind(this)
```

Comment:
Should we press `shift+enter` (DBeaver uses this)? Or anything similar with pressing `enter`? I usually look at other tools to decide. I'm just not too familiar with `alt+m` though I feel like it might works or I might be too nitpicky. Feel free to use anything that's best for you.